src/tests/ftest/util/io_utilities.py

```diff
@@ -10,8 +10,12 @@
 import time
 from logging import getLogger
 
+from command_utils import ExecutableCommand
+from command_utils_base import FormattedParameter
+from exception_utils import CommandFailure
```

Comment:
F401 'exception_utils.CommandFailure' imported but unused

Reply:
Hmm, that means the githook linting is still not working correctly.

Reply:
Yeah, the flake hook and some others still need fixing. I'm doing that here #14214